owtf/install/install.py

```diff
@@ -230,29 +118,17 @@ def danger(cls, string):
     # Path to custom scripts for tasks such as setting up/ running PostgreSQL db, run arachni, nikto, wapiti etc.
     scripts_path = os.path.join(root_dir, "scripts")
 
-    # OWTF python libraries
-    owtf_pip = os.path.join(root_dir, "install", "owtf.pip")
-
-    # Restricted tools and dictionaries which are distro independent
-    restricted_cfg = os.path.join(root_dir, "install", "distro-independent.cfg")
+    # Copy all necessary directories
+    copy_dirs('conf')
```

Comment:
These have to be either here or should be fetched from the config file. Basically it copies over `conf`, `dictionaries` and `tools` directories to `~/.owtf` so that the pip package space is not polluted :)